e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/helpers/index.ts

```diff
@@ -54,7 +54,7 @@ export const assertDashboard = ({ id, name }: { id: number; name: string }) => {
 
 type NavigateToStepOptions =
   | {
-      experience: "exploration";
+      experience: "exploration" | "metabot";
```

Comment:
I think once we have more components we want to display here, we should name these types properly. Like what is this thing that has no `resourceName`.

I think we call `exploration` a template. But I'm not even sure what a template is. Also, apparently, `metabot` is not a template. So like these inline ad-hoc types are a bit hard to grasp.

Edit: anyway, I just saw this is a type for a test helper. Maybe we don't need to do anything about it. But still, if there's a better way to categorize these types, it'd be great.